c2corg_ui/templates/area/view.html

```diff
@@ -0,0 +1,143 @@
+    delete_association_confirmation_modal, show_route_title, show_float_buttons,
+    get_document_description"/>
+<%namespace file="helpers/detailed_area_attributes.html" import="get_area_general"/>
+<%
+area_id = area['document_id']
+other_langs, missing_langs = get_lang_lists(area, lang)
+area['doctype'] = 'areas'
+%>
+
+<%block name="pagelang">lang="${lang}"</%block>
+
+<%block name="pagetitle"><title>${show_route_title(locale, True)}</title></%block>
+
+<%block name="metarobots">
+  % if version:
+    <meta name="robots" content="noindex,follow">
+  % else:
+    <meta name="robots" content="index,follow">
+  % endif
+</%block>
+
+<%block name="moduleConstantsValues">
+  module.value('mapFeatureCollection',
+  % if area['geometry']['geom_detail']:
+    {
+      "type": "FeatureCollection",
+      "properties": {},
+      "features": [
+      {
+        "type": "Feature",
+        "geometry": ${area['geometry']['geom_detail'] | n},
+        "properties": {
+          "title": ${dumps(locale['title']) | n},
+          "lang": "${locale['lang']}",
+          "documentId": ${area['document_id']},
+          "module": "areas",
+          "highlight": true
+        }
+      }
+    ]}
+  % else:
+    null
+  % endif
+  );
+
+  module.value('documentData', {
+    "document_id": ${area.get('document_id')},
+    "lang": "${lang}",
+    "type": "a",
+    "topic_id": ${dumps(locale.get('topic_id'))}
+  });
+</%block>
+
+<header class="view-details-title">
+  <h1>
+    <span class="icon-${area['area_type']} area-type" uib-tooltip="{{mainCtrl.translate('${area['area_type']}')}}"></span>
+    <span>${locale['title']}</span>
+    ${show_badge(area, 'area')}
+  </h1>
+</header>
+
+<app-map class="view-details-map"></app-map>
+
+<section class="view-details-section" app-view-details>
+
+  % if version:
+    ${show_archive_data('areas', area, locale, version)}
+  % endif
+
+  ## IMAGES
+  % if not version:
+    ${get_image_gallery()}
+  % endif
+
+  <div class="view-details-informations col-xs-12 informations">
+    <h3 class="heading informations" ng-click="mainCtrl.animateHeaderIcon($event)" data-toggle="collapse" data-target="#document-informations">
+      <span translate>Informations</span><span class="glyphicon glyphicon-menu-down"></span>
+    </h3>
+    <section id="document-informations" class="collapse in">
+      ${get_area_general(area)}
+    </section>
+  </div>
+
+  ${get_document_description(locale, 'area')}
+
+  ## ASSOCIATIONS
+  % if not version:
+    <div class="view-details-comments col-xs-12 associations">
+      <h3 class="heading associations"
+          data-toggle="collapse" data-target="#associations"
+          ng-click="mainCtrl.animateHeaderIcon($event)">
+        <span class="glyphicon glyphicon-map-marker"></span>
+        <span translate>Associated documents</span>
+        <span class="glyphicon glyphicon-menu-down"></span>
+      </h3>
+      <section id="associations" class="collapse in">
+        <p class="text-center">
```

Comment:
you should try to reproduce the styling and content of a card - the orange block are not consistent.
`div.associated-documents -> .list-item -> a -> span.list-item-title`